PalasoUIWindowsForms/PalasoUIWindowsForms.csproj

```diff
@@ -287,7 +287,7 @@
     <None Include="..\lib\common\Enchant.Net.dll.config">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
-    <Reference Include="Microsoft.VisualBasic" />
+    <Reference Include="Microsoft.VisualBasic" Condition=" '$(Configuration)' != 'DebugMono' And '$(Configuration)' != 'ReleaseMono'"/>
```

Comment:
The condition is usually OS based, not configuration based.  I don't remember the exact syntax off hand.